wgpu/Cargo.toml

```diff
@@ -76,11 +82,61 @@ git = "https://github.com/gfx-rs/naga"
 tag = "gfx-26"
 features = ["wgsl-in", "spv-out"]
 
+[[example]]
+name="boids"
+path="examples/boids/main.rs"
+test = true
+
+[[example]]
+name="bunnymark"
+path="examples/bunnymark/main.rs"
+test = true
+
+[[example]]
+name="conservative-raster"
+path="examples/conservative-raster/main.rs"
+test = true
+
+[[example]]
+name="cube"
+path="examples/cube/main.rs"
+test = true
+
 [[example]]
 name="hello-compute"
 path="examples/hello-compute/main.rs"
 test = true
 
+[[example]]
+name="mipmap"
+path="examples/mipmap/main.rs"
+test = true
+
+[[example]]
+name="msaa-line"
+path="examples/msaa-line/main.rs"
+test = true
+
+[[example]]
+name="shadow"
+path="examples/shadow/main.rs"
+test = true
+
+[[example]]
+name="skybox"
+path="examples/skybox/main.rs"
+test = true
+
+[[example]]
+name="texture-arrays"
+path="examples/texture-arrays/main.rs"
+test = true
+
+[[example]]
+name="water"
+path="examples/water/main.rs"
```

Comment:
are all these paths necessary? I'd expect these to match the default search paths

Reply:
They do, but we have to manually enable tests on every single one of them...

Reply:
Yes, I understand. So we can at least remove the `path = ` lines, it seems.